docs/api/content-tracing.md

```diff
@@ -62,14 +66,14 @@ Possible options are:
 
 The first 3 options are trace recoding modes and hence mutually exclusive.
 If more than one trace recording modes appear in the `traceOptions` string,
-the last one takes precedence. If none of the trace recording mode is specified,
-recording mode is `record-until-full`.
+the last one takes precedence. If none of the trace recording modes is
+specified, recording mode is `record-until-full`.
 
 The trace option will first be reset to the default option (record_mode set to
-`record-until-full`, enable_sampling and enable_systrace set to false)
+`record-until-full`, enable_sampling and enable_systrace set to `false`)
```

Comment:
Should all of these options be in backticks? I'm a little unclear why `record-until-full` is but `record_mode` isn't.